backend/digitales_warten.py

```diff
@@ -134,6 +134,25 @@ def add_entry(place_id, queue_id):
     db.session.commit()
     return jsonify(id=entry_id, name=entry_name, ticketNumber=ticket_number)
 
+@app.route('/places/<place_id>/queues/<queue_id>/entries/<entry_id>', methods=['DELETE'])
+def delete_entry(place_id, queue_id, entry_id):
+    place = Place.query.filter_by(id=place_id).first()
+    if place is None:
+        abort(404)
+
+    queue = Queue.query.filter_by(id=queue_id) \
+                       .filter_by(place=place).first()
```

Comment:
maybe we should just filter by `id=queue_id and place_id=place` here, we could skip the first query?

Reply:
or not, in the future we're going to need values stored in Place (like the password)

Reply:
Marked as won't fix by the author